src/routes/dictionaries/+page.svelte

```diff
@@ -125,6 +131,17 @@
                 required={isSequenceAdaptation}
               />
             </fieldset>
+          {:else if files !== undefined}
+            <fieldset class="save-dictionary-fieldset">
+              <label for="Save Dictionary">Save Command Dictionary to filesystem</label>
+              <input
+                bind:checked={persistDictionaryToFilesystem}
+                class="st-input"
+                name="Save Dictionary"
+                on:change={() => (persistDictionaryToFilesystem = !persistDictionaryToFilesystem)}
```

Comment:
You don't need to handle saving the new value to `persistDictionaryToFilesystem` because you're already binding it above. The bind is already updating the value, but this `on:change` is updating again and actually results in it effectively undoing the change.